### ISSUE TITLE
Move mongo to use local CA for SSL between nodes

### ISSUE THREAD

**Issue by lawrencegripper** [opencb/opencga#1110]

**Comment by avodovnik** [comment]
@lawrencegripper Could we expand this issue to replace _all_ SSL certs (apart from the public endpoints) to be generated with a local CA? Ideally, we'd end up only creating one CA anyway.

That would also fit well with the comment in #1026.

**Comment by jjcollinge** [comment]
A blog to help: https://www.jjcollinge.co.uk/self-signed-tls-with-terraform/

**Comment by lawrencegripper** [comment]
Yeah combined with: https://cloudinit.readthedocs.io/en/latest/topics/examples.html#configure-an-instances-trusted-ca-certificates
